tests/components/flo/test_device.py

```diff
@@ -88,13 +86,61 @@ async def test_device(
 
     assert aioclient_mock.call_count == call_count + 6
 
-    # test error sending device ping
+
+async def test_device_failures(
+    hass: HomeAssistant,
+    config_entry,
+    aioclient_mock_fixture,
+    aioclient_mock: AiohttpClientMocker,
+) -> None:
+    """Test Flo by Moen devices buffer API failures."""
+    config_entry.add_to_hass(hass)
+    assert await async_setup_component(
+        hass, FLO_DOMAIN, {CONF_USERNAME: TEST_USER_ID, CONF_PASSWORD: TEST_PASSWORD}
+    )
```

Comment:
```suggestion
    assert await hass.config_entries.async_setup(config_entry.entry_id)
```
I'm guessing it should be setup like this